backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+openapi: 3.0.0
+info:
+servers:
+paths:
+  /curation/auth/key:
+    post:
+      summary: Generate the curator's API key
+      security:
+        - cxguserCookie: []
+      responses:
```

Comment:
That's not a 403? 🤔

Reply:
Is there a reason not to regenerate the API key?

Otherwise ... 

It does not seem to be a 409 based on my reading - https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.8 - how about 400 - https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1

Reply:
based on that [link](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.8) 409 seems incorrect, but 400 also seems wrong. I believe we settle on regeneration of the API key.

Reply:
I'm fine with that. It will regenerate.

Reply:
Updated description `90ddef70`